images/linux/scripts/installers/2004/php.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+################################################################################
+##  File:  php.sh
+##  Desc:  Installs php
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
+source $HELPER_SCRIPTS/document.sh
+
+LSB_RELEASE=$(lsb_release -rs)
+
+set -e
+
+apt-add-repository ppa:ondrej/php -y
+
+# Install php7.1
+apt-fast install -y --no-install-recommends \
+    php7.1 \
+    php7.1-bcmath \
+    php7.1-bz2 \
+    php7.1-cgi \
+    php7.1-cli \
+    php7.1-common \
+    php7.1-curl \
+    php7.1-dba \
+    php7.1-dev \
+    php7.1-enchant \
+    php7.1-fpm \
+    php7.1-gd \
+    php7.1-gmp \
+    php7.1-imap \
+    php7.1-interbase \
+    php7.1-intl \
+    php7.1-json \
+    php7.1-ldap \
+    php7.1-mbstring \
+    php7.1-mcrypt \
+    php7.1-mysql \
+    php7.1-odbc \
+    php7.1-opcache \
+    php7.1-pgsql \
+    php7.1-phpdbg \
+    php7.1-pspell \
+    php7.1-readline \
+    php7.1-recode \
+    php7.1-snmp \
+    php7.1-soap \
```

Comment:
Currently, we have PHP 7.1, 7.2, 7.3, 7.4.
@alepauly , do you have an idea what versions can be safety deprecated?
For me looks like that 7.1 can be deprecated because this version is pretty long and not available on official site at all.
Not sure about 7.2, 7.3

Reply:
I think we should start with only 7.4. 
7.2 is out of support, and 7.3 will be supported until December. If we need to add one later I prefer that than building dependency on the older version unnecessarily:

https://www.php.net/supported-versions.php